website/docs/Pipelines/snM3C/README.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 1
```

Comment:
i can see this when i "display the rich diff", im not sure if we want to be able to see this?

Reply:
It won't render on the doc site. This should be ok.